os/arch/arm/src/s5j/chip/s5jt200_i2s.h

```diff
@@ -64,103 +64,103 @@
  * Pre-processor Definitions
  ************************************************************************************/
 
-#define S5J_I2S_MAXPERCLK         26000000  /* Maximum peripheral clock frequency */
```

Comment:
Revert this file changes. You revert our changes and make errors

Reply:
Benix, please check codes regarding i2s and i2c.

Reply:
I checked this, it is modified by ahreaum before

Reply:
Garg, previous version is correct. Ahreum modifed them..

Reply:
Restored previous version